pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget.go

```diff
@@ -409,6 +436,16 @@ func (c *DisruptionBudgetController) pdbForVMI(namespace, name string) (*v1beta1
 	return nil, nil
 }
 
+func migrationInProgress(vmi *virtv1.VirtualMachineInstance) bool {
+	return vmi.Status.MigrationState != nil &&
```

Comment:
Is there a special reason why we compare booleans instead of directly using their value?
```suggestion
	return vmi.Status.MigrationState != nil && !vmi.Status.MigrationState.Completed && !vmi.Status.MigrationState.Failed
```